### PR TITLE
chore(CFN): add CA & CB templated for dotnet

### PR DESCRIPTION
**Pull request by ajewellamz**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn/net/CB-Release.yml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        DbEsdkNugetAPIKeyPolicy-${ProjectName}
+      Path: /service-role/
+      PolicyDocument: !Sub |
+        {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:iam::582595803497:role/aws-crypto-tools-build-role"
```
Comment (texastony) [inline cfn/net/CB-Release.yml:150]: Question: I cannot find `582595803497` in our list of AWS Accounts we own.
Are we using another team's AWS Account?

**Review by texastony — COMMENTED** [review]
This looks like a Start for "add dotnet build configuration".
But... this is just the infrastructure.
Is this PR a WIP?

File: cfn/net/CB-Release.yml (hunk 1 above)
Comment (josecorella) [reply cfn/net/CB-Release.yml:150]: yes this account is not ours, it is the sdk net team's account

**Comment by texastony** [comment]
> Questions: @josecorella & @ajewellamz
....

Talked offline with @josecorella 
### Are we going to release DB-ESDK-DynamoDB-NET from this Repo, or a separate NET-Only repo?

A .NET specific repo will exist and will have a Release Tag, Release Notes, a README, but no Changelog.

The Changelog will live inside the Dafny repo
(which this Java repo MAY become). 
Right now, there will only be one CHANGELOG for all the runtimes.

The Language specific READMEs will detail language specific usage instructions,
but will redirect viewers to the Dafny repo to see the Source Code. 

This is what we are doing for now, we may revise it later.

File: cfn/net/CB-Release.yml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+          ]
+        }
+
+  DbEsdkNugetAPIKeyPolicy:
+    Type: "AWS::IAM::ManagedPolicy"
+    Properties:
+      ManagedPolicyName: !Sub >-
+        DbEsdkNugetAPIKeyPolicy-${ProjectName}
+      Path: /service-role/
+      PolicyDocument: !Sub |
```
Comment (texastony) [inline cfn/net/CB-Release.yml:143]:
```suggestion
      Path: /service-role/
      # 582595803497 is the sdk net team's account
      PolicyDocument: !Sub |
```

**Review by texastony — APPROVED** [review]
Left a suggested comment, otherwise LGTM

**Pull request merged**